pkg/ctl/get/nodegroup_test.go

```diff
@@ -15,12 +17,41 @@ var _ = Describe("get", func() {
 		})
 
 		It("setting --name and argument at the same time", func() {
-			cmd := newMockCmd("nodegroup", "ng", "--cluster", "dummy", "--name", "ng")
+			cmd := newMockCmd("nodegroup", "ng", "--name", "ng")
 			_, err := cmd.execute()
 			Expect(err).To(HaveOccurred())
 			Expect(err.Error()).To(ContainSubstring("Error: --name=ng and argument ng cannot be used at the same time"))
 		})
 
+		It("setting --cluster and argument at the same time", func() {
+			cmd := newMockCmd("nodegroup", "ng", "--cluster", "name")
+			_, err := cmd.execute()
+			Expect(err).To(HaveOccurred())
+			Expect(err.Error()).To(ContainSubstring("Error: --cluster=name and argument ng cannot be used at the same time"))
+		})
+
+		It("setting --name and --config-file at the same time", func() {
+			f, err := os.CreateTemp("", "configfile")
+			Expect(err).NotTo(HaveOccurred())
+			_, err = f.WriteString(configFile)
+			Expect(err).NotTo(HaveOccurred())
+			cmd := newMockCmd("nodegroup", "--name", "name", "--config-file", f.Name())
+			_, err = cmd.execute()
+			Expect(err).To(HaveOccurred())
+			Expect(err.Error()).To(ContainSubstring("Error: cannot use --name when --config-file/-f is set"))
```

Comment:
There's also `MatchError`:

```suggestion
			Expect(err).To(MatchError(ContainSubstring("Error: cannot use --name when --config-file/-f is set")))
```